### PR TITLE
feat: new tutorial landing page version

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: src/messages/en.json [hunk 1]
```diff
@@ -255,26 +349,24 @@
...
   "faq_filter.title": "Products",
-  "faq_page.results_found": "results found",
   "sitemap_page.title": "Sitemap",
   "sitemap_page_section_additional_resources.title": "Additional resources",
   "sitemap_page_section_additional_resources.support_rules": "Support Rules",
   "sitemap_page_section_additional_resources.known_issues": "Known Issues",
   "sitemap_page_section_additional_resources.faq": "FAQ",
   "sitemap_page_section_additional_resources.announcements": "Announcements",
-  "sitemap_page_section_additional_resources.dev_portal": "Developer portal",
+  "sitemap_page_section_additional_resources.dev_portal": "Developers Portal",
```
Comment (julia-rabello) [inline src/messages/en.json:366]:
```suggestion
  "sitemap_page_section_additional_resources.dev_portal": "Developer Portal",
```

File: src/messages/es.json [hunk 2]
```diff
@@ -7,9 +7,9 @@
   "landing_page_documentation.title": "Documentation",
   "documentation_start_here.title": "Comece aqui",
   "documentation_start_here.description": "Aprenda, passo a passo, tudo que precisa saber para começar a usar os principais módulos da VTEX.",
-  "documentation_tutorials.title": "Tutoriais",
-  "documentation_tutorials.description": "Acesse a documentação de todos os módulos da plataforma e tire suas dúvidas.",
-  "documentation_developers_portal.title": "Portal del desarrollador",
```
Comment (julia-rabello) [inline src/messages/es.json:12]: why delete this?

File: src/messages/es.json [hunk 3]
```diff
@@ -120,9 +120,6 @@
   "start_here_page_modules.content.10.title": "Desarrollo de la tienda",
   "start_here_page_modules.content.10.description": "Conozca los fundamentos del desarrollo de tiendas VTEX.",
   "start_here_page_modules.content.10.link": "tracks/frontend",
-  "start_here_page_modules.content.11.title": "Datos y privacidad",
-  "start_here_page_modules.content.11.description": "Conozca nuestro compromiso con los datos personales y la privacidad en VTEX.",
-  "start_here_page_modules.content.11.link": "tracks/datos-personales",
```
Comment (julia-rabello) [inline src/messages/es.json:125]: please restore these lines

File: src/messages/es.json [hunk 4]
```diff
@@ -195,6 +192,103 @@
   "start_here_page_conversational.content.0.title": "VTEX Assisted Sales (SuiteShare)",
   "start_here_page_conversational.content.0.description": "Aprenda soluciones para atraer más clientes, organizar los servicios de su equipo, captar datos de clientes, cerrar ventas y medir resultados.",
   "start_here_page_conversational.content.0.link": "tracks/que-es-suiteshare",
+  "start_here_tutorials_page.title": "Tutoriales hub",
```
Comment (julia-rabello) [inline src/messages/es.json:195]:
```suggestion
  "start_here_tutorials_page.title": "Tutoriales",
```

File: src/messages/es.json [hunk 5]
```diff
@@ -226,16 +320,16 @@
   "support_status.title": "Status",
   "support_status.description": "Vea el registro del historial de la plataforma.",
   "support_plans.title": "Planes de soporte",
-  "support_plans.link": "/faq/como-funciona-el-soporte-de-vtex",
```
Comment (julia-rabello) [inline src/messages/es.json:229]: please restore this line

File: src/messages/es.json [hunk 6]
```diff
@@ -226,16 +320,16 @@
...
   "support_plans.description": "Consulte los detalles de los planes de soporte disponibles para cada plan comercial.",
+  "announcement_bar.headline": "📢 Queremos saber más sobre ti y cómo utilizas nuestra documentación. ",
+  "announcement_bar.button": "¡Complete nuestra encuesta! Toma menos de 5 minutos.",
```
Comment (julia-rabello) [inline src/messages/es.json:325]: this was removed before. why add it back?

File: src/messages/es.json [hunk 7]
```diff
@@ -226,16 +320,16 @@
...
   "documentation_reading_time.text": "{minutes} min de lectura",
   "sidebar_known_issues.description": "Accede a contenido y encuentra soluciones a todas tus preguntas sobre la plataforma.",
   "sidebar_support_rules.description": "Vea los detalles de los planes de soporte disponibles para cada plan comercial.",
   "updates_info_notes.title": "Preguntas Frecuentes",
   "updates_info_notes.description": "Encuentra respuestas a tus preguntas clave de manera rápida y práctica.",
   "menu_status.description": "Seguir el registro histórico de la plataforma.",
   "sidebar_support_rules.title": "Directrices de Soporte",
-  "sidebar_support_rules.link": "/faq/como-funciona-el-soporte-de-vtex",
```
Comment (julia-rabello) [inline src/messages/es.json:238]: please restore this line

File: src/messages/es.json [hunk 8]
```diff
@@ -258,26 +352,24 @@
   "known_issues_result.empty": "Ningún resultado coincidente. Pruebe con otros filtros.",
   "known_issues_date.created": "Creado en",
   "known_issues_date.updated": "Actualizado en",
-  "tutorial_and_solutions_page.title": "Tutoriales y Soluciones",
-  "tutorial_and_solutions_page.description": "Accede a la documentación de todos los módulos de la plataforma y resuelve tus dudas.",
+  "tutorials_page.title": "Tutoriales hub",
```
Comment (julia-rabello) [inline src/messages/es.json:355]:
```suggestion
  "tutorials_page.title": "Tutoriales",
```

File: src/messages/es.json [hunk 9]
```diff
@@ -258,26 +352,24 @@
...
+  "tutorials_page.description": "Navegue por nuestros tutoriales para mejorar su comprensión de la plataforma.",
   "date_text.created": "Publicado en",
   "date_text.updated": "Última actualización",
   "announcements_page_result.empty": "Ningún resultado coincidente.",
   "announcements_page_search.placeholder": "Buscar en Anuncios...",
   "search_result.empty": "Ningún resultado coincidente. Pruebe con otros filtros.",
   "faq_filter.title": "Produtos",
-  "faq_page.results_found": "resultados encontrados",
```
Comment (julia-rabello) [inline src/messages/es.json:269]: please restore this line

File: src/messages/es.json [hunk 10]
```diff
@@ -258,26 +352,24 @@
...
   "sitemap_page.title": "Mapa del sitio",
   "sitemap_page_section_additional_resources.title": "Recursos adicionales",
   "sitemap_page_section_additional_resources.support_rules": "Directrices de Soporte",
   "sitemap_page_section_additional_resources.known_issues": "Problemas Conocidos",
   "sitemap_page_section_additional_resources.faq": "Preguntas Frecuentes",
   "sitemap_page_section_additional_resources.announcements": "Anuncios",
-  "sitemap_page_section_additional_resources.dev_portal": "Portal de desarrolladores",
+  "sitemap_page_section_additional_resources.dev_portal": "Portal de Desarrolladores",
```
Comment (julia-rabello) [inline src/messages/es.json:369]: why change this?

File: src/messages/es.json [hunk 11]
```diff
@@ -258,26 +352,24 @@
...
   "sitemap_page_section_additional_resources.support": "Ayuda",
   "sitemap_page_section_additional_resources.community": "Comunidad",
   "sitemap_page_section_additional_resources.feedback": "Comentarios",
-  "chip.all_results": "Todos resultados",
```
Comment (julia-rabello) [inline src/messages/es.json:280]: please restore this line

File: src/messages/pt.json [hunk 12]
```diff
@@ -7,20 +7,20 @@
   "landing_page_documentation.title": "Documentação",
   "documentation_start_here.title": "Comece aqui",
   "documentation_start_here.description": "Aprenda, passo a passo, tudo que precisa saber para começar a usar os principais módulos da VTEX.",
-  "documentation_tutorials.title": "Tutoriais",
-  "documentation_tutorials.description": "Acesse a documentação de todos os módulos da plataforma e tire suas dúvidas.",
-  "documentation_developers_portal.title": "Portal do desenvolvedor",
+  "documentation_tutorials.title": "Tutoriais hub",
```
Comment (julia-rabello) [inline src/messages/pt.json:10]:
```suggestion
  "documentation_tutorials.title": "Tutoriais",
```

File: src/messages/pt.json [hunk 13]
```diff
@@ -7,20 +7,20 @@
...
+  "documentation_tutorials.description": "Navegue pelos nossos tutoriais para melhorar sua compreensão da plataforma.",
+  "documentation_developers_portal.title": "Portal do Desenvolvedor",
```
Comment (julia-rabello) [inline src/messages/pt.json:12]: why change this line? this had already been reviewed by loc

File: src/messages/pt.json [hunk 14]
```diff
@@ -7,20 +7,20 @@
...
   "documentation_developers_portal.description": "Visite nosso portal de desenvolvedores e encontre informações técnicas sobre integrações, APIs e recursos de desenvolvimento.",
-  "landing_page_faq.title": "Perguntas frequentes",
+  "landing_page_faq.title": "Perguntas Frequentes",
```
Comment (julia-rabello) [inline src/messages/pt.json:14]: why change this?

File: src/messages/pt.json [hunk 15]
```diff
@@ -7,20 +7,20 @@
...
   "landing_page_faq.description": "Encontre respostas para dúvidas mais frequentes de forma rápida e prática.",
   "landing_page_faq.button": "Ver todas as perguntas",
-  "faq_page_search.placeholder": "Pesquise por Perguntas frequentes...",
+  "faq_page_search.placeholder": "Pesquise por Perguntas Frequentes...",
```
Comment (julia-rabello) [inline src/messages/pt.json:17]: why change this?

File: src/messages/pt.json [hunk 16]
```diff
@@ -7,20 +7,20 @@
...
   "faq_order_error.title": "Erros de integração de pedidos de marketplaces",
   "faq_order_error.description": "Entenda como identificar erros em pedidos externos.",
   "faq_handling.title": "Por que meu pedido está parado em “Preparando entrega”?",
-  "faq_handling.description": "“Preparando entrega” é um status do fluxo de pedidos em que se espera o cadastro da nota fiscal para a fatura.",
+  "faq_handling.description": "Preparando Entrega é um status do fluxo de pedidos em que se espera o cadastro da nota fiscal para a fatura.",
```
Comment (julia-rabello) [inline src/messages/pt.json:21]:
```suggestion
  "faq_handling.description": "'Preparando entrega' é um status do fluxo de pedidos em que se espera o cadastro da nota fiscal para a fatura.",
```

File: src/messages/pt.json [hunk 17]
```diff
@@ -7,20 +7,20 @@
...
   "faq_product_visible.title": "Por que o produto não aparece no site?",
-  "faq_product_visible.description": "Para um produto aparecer disponível na loja, ele precisa ser atendido pela logística configurada na loja.",
+  "faq_product_visible.description": "Para um produto aparecer disponível na loja ele precisa ser atendido pela logística configurada na loja.",
```
Comment (julia-rabello) [inline src/messages/pt.json:23]:
```suggestion
  "faq_product_visible.description": "Para um produto aparecer disponível na loja, ele precisa ser atendido pela logística configurada na loja.",
```

File: src/messages/pt.json [hunk 18]
```diff
@@ -32,7 +32,7 @@
   "landing_page_announcements.button": "Acesse todas as novidades",
   "dashboards.description": "Manage data from your unified commerce",
   "orders.description": "Check the status of orders placed in your store.",
-  "landing_page_education.title": "Ainda tem dúvidas?",
+  "landing_page_education.title": "Ainda possui dúvidas?",
```
Comment (julia-rabello) [inline src/messages/pt.json:35]:
```suggestion
  "landing_page_education.title": "Ainda tem dúvidas?",
```

File: src/messages/pt.json [hunk 19]
```diff
@@ -46,7 +46,7 @@
   "landing_page_education_community.description": "Encontre soluções e compartilhe ideias na comunidade VTEX.",
   "landing_page_education_community.textLink": "Junte-se à nossa comunidade",
   "landing_page_support.title": "Recursos de suporte",
-  "landing_page_footer_developer_portal.message": "Developer portal",
+  "landing_page_footer_developer_portal.message": "Developers Portal",
```
Comment (julia-rabello) [inline src/messages/pt.json:49]:
```suggestion
  "landing_page_footer_developer_portal.message": "Developer Portal",
```

File: src/messages/pt.json [hunk 20]
```diff
@@ -120,9 +120,6 @@
   "start_here_page_modules.content.10.title": "Desenvolvimento de Loja",
   "start_here_page_modules.content.10.description": "Aprenda os fundamentos de desenvolvimento de loja na VTEX.",
   "start_here_page_modules.content.10.link": "tracks/frontend",
-  "start_here_page_modules.content.11.title": "Dados e privacidade",
-  "start_here_page_modules.content.11.description": "Conheça a abordagem da VTEX em relação a dados pessoais e privacidade.",
-  "start_here_page_modules.content.11.link": "tracks/dados-pessoais",
```
Comment (julia-rabello) [inline src/messages/pt.json:125]: please restore these lines

File: src/messages/pt.json [hunk 21]
```diff
@@ -210,6 +207,103 @@
   "start_here_page_conversational.content.0.title": "VTEX Assisted Sales (SuiteShare)",
   "start_here_page_conversational.content.0.description": "Aprenda soluções para atrair mais clientes, organizar os atendimentos do seu time, capturar dados dos clientes, fechar vendas e mensurar resultados.",
   "start_here_page_conversational.content.0.link": "tracks/o-que-e-suiteshare",
+  "start_here_tutorials_page.title": "Tutoriais hub",
```
Comment (julia-rabello) [inline src/messages/pt.json:210]:
```suggestion
  "start_here_tutorials_page.title": "Tutoriais",
```

File: src/messages/pt.json [hunk 22]
```diff
@@ -241,21 +335,21 @@
   "support_status.title": "Status",
   "support_status.description": "Acompanhe o status da plataforma e saiba o histórico de instabilidade.",
   "support_plans.title": "Planos de suporte",
-  "support_plans.link": "/faq/como-funciona-o-suporte-da-vtex",
   "support_plans.description": "Veja detalhes dos planos de suporte disponíveis para cada plano comercial.",
+  "announcement_bar.headline": "📢 Queremos saber mais sobre você e como você utiliza a nossa documentação. ",
+  "announcement_bar.button": "Participe de nossa pesquisa! Leva menos de 5 minutos.",
```
Comment (julia-rabello) [inline src/messages/pt.json:340]: this was deleted before. why bring it back?

File: src/messages/pt.json [hunk 23]
```diff
@@ -241,21 +335,21 @@
   "support_status.title": "Status",
   "support_status.description": "Acompanhe o status da plataforma e saiba o histórico de instabilidade.",
   "support_plans.title": "Planos de suporte",
-  "support_plans.link": "/faq/como-funciona-o-suporte-da-vtex",
```
Comment (julia-rabello) [inline src/messages/pt.json:244]: please restore this line

File: src/messages/pt.json [hunk 24]
```diff
@@ -241,21 +335,21 @@
...
   "support_plans.description": "Veja detalhes dos planos de suporte disponíveis para cada plano comercial.",
+  "announcement_bar.headline": "📢 Queremos saber mais sobre você e como você utiliza a nossa documentação. ",
+  "announcement_bar.button": "Participe de nossa pesquisa! Leva menos de 5 minutos.",
   "documentation_reading_time.text": "{minutes} min de leitura",
   "sidebar_known_issues.description": "Acesse conteúdo e encontre soluções para todas as suas dúvidas sobre a plataforma.",
   "sidebar_support_rules.description": "Veja detalhes dos planos de suporte disponíveis para cada plano comercial.",
-  "updates_info_notes.title": "Perguntas frequentes",
+  "updates_info_notes.title": "Perguntas Frequentes",
```
Comment (julia-rabello) [inline src/messages/pt.json:344]: why change this?

File: src/messages/pt.json [hunk 25]
```diff
@@ -241,21 +335,21 @@
...
   "updates_info_notes.description": "Encontre respostas para suas principais dúvidas de forma rápida e prática.",
   "menu_status.description": "Acompanhe o registro do histórico de plataforma.",
   "sidebar_support_rules.title": "Diretivas de suporte",
-  "sidebar_support_rules.link": "/faq/como-funciona-o-suporte-da-vtex",
-  "sidebar_known_issues.title": "Problemas conhecidos",
+  "sidebar_known_issues.title": "Problemas Conhecidos",
```
Comment (julia-rabello) [inline src/messages/pt.json:348]: why change this? please revert

File: src/messages/pt.json [hunk 26]
```diff
@@ -241,21 +335,21 @@
...
   "menu_status.title": "Status",
   "known_issues_page.title": "Problemas conhecidos",
   "known_issues_page.subtitle": "Encontre todos os problemas conhecidos e saiba seu status e soluções.",
-  "known_issues_page_search.placeholder": "Pesquise por Problemas conhecidos...",
+  "known_issues_page_search.placeholder": "Pesquise por Problemas Conhecidos...",
```
Comment (julia-rabello) [inline src/messages/pt.json:352]:
```suggestion
  "known_issues_page_search.placeholder": "Pesquise por Problemas conhecidos...",
```

File: src/messages/pt.json [hunk 27]
```diff
@@ -273,26 +367,24 @@
...
-  "tutorial_and_solutions_page.description": "Acesse a documentação de todas os módulos da plataforma e tire suas dúvidas.",
+  "tutorials_page.title": "Tutoriais hub",
+  "tutorials_page.description": "Navegue pelos nossos tutoriais para melhorar sua compreensão da plataforma.",
   "date_text.created": "Publicado em",
   "date_text.updated": "Última atualização em",
   "announcements_page_result.empty": "Nenhum resultado correspondente.",
   "announcements_page_search.placeholder": "Pesquise em Comunicados...",
   "search_result.empty": "Nenhum resultado correspondente. Tente outros filtros.",
   "faq_filter.title": "Produtos",
-  "faq_page.results_found": "resultados encontrados",
```
Comment (julia-rabello) [inline src/messages/pt.json:284]: please restore this line

File: src/messages/pt.json [hunk 28]
```diff
@@ -273,26 +367,24 @@
...
   "sitemap_page.title": "Mapa do site",
   "sitemap_page_section_additional_resources.title": "Recursos Adicionais",
   "sitemap_page_section_additional_resources.support_rules": "Diretrizes de Suporte",
-  "sitemap_page_section_additional_resources.known_issues": "Problemas conhecidos",
-  "sitemap_page_section_additional_resources.faq": "Perguntas frequentes",
+  "sitemap_page_section_additional_resources.known_issues": "Problemas Conhecidos",
+  "sitemap_page_section_additional_resources.faq": "Perguntas Frequentes",
   "sitemap_page_section_additional_resources.announcements": "Comunicados",
-  "sitemap_page_section_additional_resources.dev_portal": "Developer portal",
+  "sitemap_page_section_additional_resources.dev_portal": "Developers Portal",
```
Comment (julia-rabello) [inline src/messages/pt.json:384]: None of this should be changed

File: src/messages/pt.json [hunk 29]
```diff
@@ -273,26 +367,24 @@
...
   "sitemap_page_section_additional_resources.support": "Suporte",
   "sitemap_page_section_additional_resources.community": "Comunidade",
   "sitemap_page_section_additional_resources.feedback": "Feedback",
-  "chip.all_results": "Todos resultados",
```
Comment (julia-rabello) [inline src/messages/pt.json:295]: please restore

File: src/messages/en.json [hunk 30]
```diff
@@ -223,16 +317,16 @@
   "support_status.title": "Status",
   "support_status.description": "See the platform history log.",
   "support_plans.title": "Support plans",
-  "support_plans.link": "/faq/how-does-vtex-support-work",
   "support_plans.description": "See details of support plans available for each commercial plan.",
+  "announcement_bar.headline": "📢 We want to know more about you and how you use our docs. ",
+  "announcement_bar.button": "Fill in our survey! It takes less than 5 minutes.",
```
[the remark this answers is not in the document]
Comment (PedroAntunesCosta) [reply src/messages/en.json:322]: Maybe the branch was created before we merged this removal. Anyway, I think this should be removed.

File: src/messages/en.json (hunk 1 above)
Comment (PedroAntunesCosta) [reply src/messages/en.json:366]:
```suggestion
  "sitemap_page_section_additional_resources.dev_portal": "Developer portal",
```

File: src/messages/es.json [hunk 31]
```diff
@@ -46,7 +46,7 @@
   "landing_page_education_community.description": "Encuentra solucciones y comparte ideas en la comunidad de VTEX.",
   "landing_page_education_community.textLink": "Únete a nuestra comunidad",
   "landing_page_support.title": "Recursos de soporte",
-  "landing_page_footer_developer_portal.message": "Developer portal",
+  "landing_page_footer_developer_portal.message": "Developers Portal",
```
[the remark this answers is not in the document]
Comment (PedroAntunesCosta) [reply src/messages/es.json:49]:
```suggestion
  "landing_page_footer_developer_portal.message": "Developer portal",
```

File: src/messages/es.json (hunk 6 above)
Comment (PedroAntunesCosta) [reply src/messages/es.json:325]: Maybe the branch was created before we merged this removal. Anyway, I agree this should be removed.

File: src/messages/pt.json (hunk 15 above)
Comment (PedroAntunesCosta) [reply src/messages/pt.json:17]: I think many of this weird string changes are due to not updating from main. Many of these I recognize from the PR I merged with sentence case fixes and feedbacks from localization.

**Comment by PedroAntunesCosta** [comment]
@julia-rabello I think many of this weird string changes are due to not updating an old branch from main. Many of these I recognize from the PR I merged with sentence case fixes and feedbacks from localization.

**Review by PedroAntunesCosta — CHANGES_REQUESTED** [review]
@ricardoaerobr in addition to the comments Júlia made above, I would like to point, as we talked earlier in our sync, that the links in the cards don't work. They point to a 404 tutorial. I consider that, given we already know the slugs we want each card to point to, a page with placeholders is not ready for code review.

Please add the slugs to each card, corresponding to the category slug found in `public/navigation.json`.

**Comment by PedroAntunesCosta** [comment]
Also, please add a release label. I believe `release-minor` is appropriate.

File: src/messages/en.json (hunk 30 above)
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply src/messages/en.json:322]: Exactly! Removed!

File: src/messages/es.json (hunk 6 above)
Comment (ricardoaerobr) [reply src/messages/es.json:325]: Exactly! Removed!

File: src/messages/es.json (hunk 10 above)
Comment (ricardoaerobr) [reply src/messages/es.json:369]: This change is due to another modification occurring after the creation of this branch. I will restore the original information.

File: src/messages/pt.json (hunk 13 above)
Comment (ricardoaerobr) [reply src/messages/pt.json:12]: This change is due to another modification occurring after the creation of this branch. I will restore the original information.

File: src/messages/pt.json (hunk 14 above)
Comment (ricardoaerobr) [reply src/messages/pt.json:14]: This change is due to another modification occurring after the creation of this branch. I will restore the original information.

File: src/messages/pt.json (hunk 15 above)
Comment (ricardoaerobr) [reply src/messages/pt.json:17]: This change is due to another modification occurring after the creation of this branch. I will restore the original information.

File: src/messages/pt.json (hunk 24 above)
Comment (ricardoaerobr) [reply src/messages/pt.json:344]: This change is due to another modification occurring after the creation of this branch. I will restore the original information.

File: src/messages/pt.json (hunk 25 above)
Comment (ricardoaerobr) [reply src/messages/pt.json:348]: This change is due to another modification occurring after the creation of this branch. I will restore the original information.

File: src/messages/pt.json (hunk 28 above)
Comment (ricardoaerobr) [reply src/messages/pt.json:384]: This change is due to another modification occurring after the creation of this branch. I will restore the original information.

File: src/messages/pt.json [hunk 32]
```diff
@@ -8,7 +8,7 @@
   "documentation_start_here.title": "Comece aqui",
   "documentation_start_here.description": "Aprenda, passo a passo, tudo que precisa saber para começar a usar os principais módulos da VTEX.",
   "documentation_tutorials.title": "Tutoriais",
-  "documentation_tutorials.description": "Acesse a documentação de todos os módulos da plataforma e tire suas dúvidas.",
+  "documentation_tutorials.description": "Navegue pelos nossos tutoriais para melhorar sua compreensão da plataforma.",
```
Comment (julia-rabello) [inline src/messages/pt.json:11]: "melhorar a compreensão" parece estranho

File: src/messages/pt.json [hunk 33]
```diff
@@ -273,8 +338,8 @@
   "known_issues_result.empty": "Nenhum resultado correspondente. Tente outros filtros.",
   "known_issues_date.created": "Criado em",
   "known_issues_date.updated": "Atualizado em",
-  "tutorial_and_solutions_page.title": "Tutoriais",
-  "tutorial_and_solutions_page.description": "Acesse a documentação de todas os módulos da plataforma e tire suas dúvidas.",
+  "tutorials_page.title": "Tutoriais",
+  "tutorials_page.description": "Navegue pelos nossos tutoriais para melhorar sua compreensão da plataforma.",
```
Comment (julia-rabello) [inline src/messages/pt.json:342]: mesma coisa aqui

**Review by julia-rabello — CHANGES_REQUESTED** [review]
- Card links are still wrong. They should not be `tutorials/{categoryName}`. They should be `{docs/tutorial/{categoryName}`
- I noticed not all capitalization adjustments were rolled back / synced with main (e.g. `Portal del desarrollador` was changed to `Portal del Desarrolador`)

File: src/messages/pt.json (hunk 32 above)
Comment (ricardoaerobr) [reply src/messages/pt.json:11]: Navegue pelos nossos tutoriais para melhorar seu conhecimento sobre a plataforma

File: src/messages/pt.json (hunk 33 above)
Comment (ricardoaerobr) [reply src/messages/pt.json:342]: Navegue pelos nossos tutoriais para melhorar seu conhecimento sobre a plataforma

**Review by PedroAntunesCosta — CHANGES_REQUESTED** [review]
The card links are wrong.

```
https://deploy-preview-134--leafy-mooncake-7c2e5e.netlify.app/pt/docs/docs/tutorial/produtos-beta
```

File: src/messages/pt.json [hunk 34]
```diff
@@ -250,7 +316,6 @@
   "updates_info_notes.description": "Encontre respostas para suas principais dúvidas de forma rápida e prática.",
   "menu_status.description": "Acompanhe o registro do histórico de plataforma.",
   "sidebar_support_rules.title": "Diretivas de suporte",
-  "sidebar_support_rules.link": "/faq/como-funciona-o-suporte-da-vtex",
```
Comment (julia-rabello) [inline src/messages/pt.json:253]: please restore this line

**Review by julia-rabello — CHANGES_REQUESTED** [review]
Could we use category instead of cat (both in navigation.json and messages)? Looking at the result, it would be weird to have slugs like cat-apps, cat-sellers, cat-shipping 🐱

File: src/messages/pt.json (hunk 34 above)
Comment (ricardoaerobr) [reply src/messages/pt.json]: Done!

**Review by PedroAntunesCosta — CHANGES_REQUESTED** [review]
It is not working, I am getting `500` in the category pages.
image

File: src/pages/sitemap/index.tsx [hunk 35]
```diff
@@ -22,7 +22,7 @@ const SiteMapPage: Page<Props> = ({ sections }) => {
 
   const documentationTitleTranslated: { [key: string]: string } = {
     'Start here': intl.formatMessage({ id: 'documentation_start_here.title' }),
-    'Tutorials & Solutions': intl.formatMessage({
+    Tutorials: intl.formatMessage({
```
Comment (julia-rabello) [inline src/pages/sitemap/index.tsx:25]: shouldn't you keep Tutorials between ' ' like the line before?

**Comment by julia-rabello** [comment]
@ricardoaerobr sorry, I think you were correct before not using Tutorials between ' '. Now the deploy fails with this error:

image

**Review by julia-rabello — APPROVED** [review]

**Pull request merged**